src/config/reviews.ts

```diff
@@ -1,4 +1,3 @@
 export const enableReviewsIo = true;
-export const reviewsIoReviewsPerPage = 3;
 export const enableTrustpilot = false;
```

Comment:
One idea, for your consideration. We use an enum for the reviews provider, `REVIEWS_IO` and `TRUSTPILOT`. We then have a single `getProductReviews` fn which switches on the enum. This should eliminate the need to comment out some of the transform code below.

We could easily push the enum out to an env var, which is a little bit less that the user will actually need to touch the codebase.

Reply:
The problem is how to handle the types when the fragment disappears from the query. I think we should get this iteration in and then game plan for a better way to approach this.